apps/angular/src/app/features/auth/log-in/log-in.component.ts

```diff
@@ -0,0 +1,62 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, DestroyRef } from '@angular/core';
+import { Router } from '@angular/router';
+import { EMPTY, catchError } from 'rxjs';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { AuthService } from '@js-camp/angular/core/services/auth.service';
+
+/** Log in component. */
+@Component({
+	selector: 'camp-log-in',
+	templateUrl: './log-in.component.html',
+	styleUrls: ['./log-in.component.css'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LoginComponent {
+	/** Log in form. */
+	protected loginForm: FormGroup;
```

Comment:
Looks like we are not mutating `logingForm` field and can make it read-only
```suggestion
	protected readonly loginForm: FormGroup;
```